fix-vtex-connector-v2-pix_pt (#1149)
mercadopago/devsite-docs: guides/plugins/unofficial/vtex.pt.md

VTEX é uma **plataforma e-commerce para sua loja virtual**, que permite processar pagamentos através do Mercado Pago.

> Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) e [Pix](#bookmark_condição_de_pagamento_Pix) no próprio checkout da sua loja ou [redirecionado para o site do Mercado Pago](#bookmark_condição_de_pagamento_checkout_pro).

Reviewer: ```suggestion
Você poderá oferecer aos seus clientes a possibilidade de pagar com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) e [Pix](#bookmark_condição_de_pagamento_pix) no próprio checkout da sua loja ou [redirecionado para o site do Mercado Pago](#bookmark_condição_de_pagamento_checkout_pro).
```

3. Configure as condições de pagamento Mercado Pago.
4. Configure device fingerprint.

## Cadastre uma afiliações de gateway

Uma afiliação de gateway é um conjunto de configurações que representam o processamento de seus pagamentos com o Mercado Pago.
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) ou [Checkout Pro](#bookmark_condição_de_pagamento_checkout_pro), e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_Pix).

### Criando afiliação de gateway MercadoPagoV1

Para criar uma **afiliação de gateway de pagamento com o Mercado Pago**, siga os passos abaixo:

1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.

> 2. Na aba Afiliações de Gateways, clique no botão "+".

Reviewer: ```suggestion
2. Na aba "Afiliações de Gateways", clique no botão "+".
```

3. Configure as condições de pagamento Mercado Pago.
4. Configure device fingerprint.

## Cadastre uma afiliações de gateway

Uma afiliação de gateway é um conjunto de configurações que representam o processamento de seus pagamentos com o Mercado Pago.

> Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).

Reviewer: ```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).
```

> Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) ou [Checkout Pro](#bookmark_condição_de_pagamento_checkout_pro), e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_Pix).

Reviewer: ```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_pix).
```

> Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com [cartão de crédito](#bookmark_condição_de_pagamento_cartão_de_crédito), [boleto bancário](#bookmark_condição_de_pagamento_boleto_bancário) ou [Checkout Pro](#bookmark_condição_de_pagamento_checkout_pro), e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)** para o processamento de pagamento [Pix](#bookmark_condição_de_pagamento_Pix).

Reviewer 2: ```suggestion
Você poderá utilizar a **afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1)**, onde possibilita processar pagamentos com cartão de crédito, boleto bancário ou Checkout Pro, e também adicionar a **afiliação de gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)** para o processamento de pagamento Pix.
```

E pronto! Sua afiliação com o Mercado Pago já está ativa!

> ### Criando afiliação de gateway MercadoPagoV2 com Pix

Reviewer: @abderossi ¿no dijimos de sacarlo al "com Pix"?

> ### Requisito prévio

Reviewer: ```suggestion
#### Requisito prévio
```

#### &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Obtenha uma chave Pix

Para começar, você deve ter cadastrada uma chave Pix na conta do vendedor. Este dado é único, serve para identificar sua conta e permitirá que você utilize as funcionalidades do meio de pagamento.

> [Conheça como criar uma chave Pix](https://www.mercadopago.com.br/ajuda/17843)

Reviewer: ```suggestion
[Conheça como criar uma chave Pix](https://www.mercadopago.com.br/stop/pix?url=https%3A%2F%2Fwww.mercadopago.com.br%2Fadmin-pix-keys%2Fmy-keys&authentication_mode=required)
```

> Considere que o registro da chave Pix pode levar alguns minutos.

Para criar uma **afiliação de gateway de pagamento Pix com o MercadoPagoV2**, siga os passos abaixo:

1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.

> 2. Na aba Afiliações de Gateways, clique no botão "+".

Reviewer: ```suggestion
2. Na aba "Afiliações de Gateways", clique no botão "+".
```

3. Clique no conector MercadoPagoV1.
4. Preencha os campos correspondentes e clique em "Salvar".

| Campos | Dados necessários |
| --- | --- |
| Application Key | Refere-se às suas [credenciais]([FAKER][CREDENTIALS][URL]) do Mercado Pago. Complete com sua Public Key. |
| Application Token | Refere-se às suas [credenciais]([FAKER][CREDENTIALS][URL]) do Mercado Pago. Complete com seu Access Token. |

![Setting affiliation in VTEX](/images/vtex/vtex_afiliacao_gateway_v2.gif)

E pronto! Sua afiliação com o Mercado Pago já está ativa!

> WARNING
>
> Importante
>

> > Lembre-se que o gateway [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix) por enquanto adiciona apenas o Pix como meio de pagamento. Para configurar outros métodos de pagamento, crie uma afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1).

Reviewer: ```suggestion
> Lembre-se que o gateway MercadoPagoV2 por enquanto adiciona apenas o Pix como meio de pagamento. Para configurar outros métodos de pagamento, crie uma afiliação de gateway [MercadoPagoV1](#bookmark_criando_afiliação_de_gateway_mercadopagov1).
```

### Condição de pagamento boleto bancário

Para criar uma **condição de pagamento boleto bancário utilizando sua afiliação com o Mercado Pago**, siga os passos abaixo:

1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
2. Na aba "Condições de pagamento", clique no botão "+".
3. Na seção "Boleto", escolha "Boleto Bancário" para adicionar ao meio de pagamento.
4. Informe o "nome da regra" para facilitar a identificação e ative a condição de pagamento no campo "Status".
5. Informe sua afiliação com o MercadoPagoV1, no campo "Processar com a afiliação".
6. Clique em "Salvar".
<p>&nbsp;</p>

![Setting payments in VTEX](/images/vtex/vtex_condicao_pagamento_boleto_3.gif)

### Condição de pagamento Pix

> Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, siga os passos abaixo:

Reviewer: ```suggestion
Para **criar uma condição de pagamento Pix utilizando sua afiliação com o [MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2)**, siga os passos abaixo:
```

1. No painel de administração de sua loja, acesse "Configurações" do módulo de pagamentos.
2. Na aba "Condições de pagamento", clique no botão "+".

> 3. Na seção "Boleto", escolha "Boleto Bancário" para adicionar ao meio de pagamento.

Reviewer: ```suggestion
3. Na seção “Pagamento Instantâneo”, escolha o meio de pagamento Pix.
```

4. Informe o "nome da regra" para facilitar a identificação e ative a condição de pagamento no campo "Status".

> 5. Informe sua afiliação com o **[MercadoPagoV2](#bookmark_criando_afiliação_de_gateway_mercadopagov2_com_pix)**, no campo "Processar com a afiliação".

Reviewer: ```suggestion
5. Informe sua afiliação com o MercadoPagoV2, no campo "Processar com a afiliação".
```

6. Clique em "Salvar".
<p>&nbsp;</p>

![Setting payments in VTEX](/images/vtex/vtex_condicao_pagamento_pix.gif)

### Condição de pagamento Checkout Pro

> O [Checkout Pro](https://www.mercadopago.com.br/developers/pt/guides/online-payments/checkout-pro/introduction) é a integração que te permite cobrar através do Mercado Pago, contando com as meios de pagamentos como Pix, boleto parcelado, débito virtual da Caixa e carteira digital do Mercado Pago.

Reviewer: ```suggestion
O [Checkout Pro](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/pt/guides/online-payments/checkout-pro/introduction) é a integração que te permite cobrar através do Mercado Pago, contando com as meios de pagamentos como Pix, boleto parcelado, débito virtual da Caixa e carteira digital do Mercado Pago.
```

>

> > Caso as condições de pagamento MercadoPagoPro e MercadoPagoWallet não estiverem disponíveis para adicioná-las, certifique em sua loja de aplicativos da VTEX que o App Mercado Pago Payment APP esteja instalado ou solicite a instalação pela equipe VTEX sob demanda através de ticket no [Support VTEX](https://support.vtex.com/). <br>

Reviewer: ```suggestion
> Caso as condições de pagamento MercadoPagoPro e MercadoPagoWallet não estiverem disponíveis para adicioná-las, certifique em sua loja de aplicativos da VTEX que o App Mercado Pago Payment APP esteja instalado ou solicite a instalação pela equipe VTEX sob demanda através de ticket no [Support VTEX](https://support.vtex.com). <br>
```

## Configurando o parcelamento em sua conta Mercado Pago

> 1. Acesse sua [conta Mercado Pago](https://www.mercadopago.com.br/business/) e clique em "Seu negócio"

Reviewer: ```suggestion
1. Acesse sua [conta Mercado Pago](https://www.mercadopago[FAKER][URL][DOMAIN]/business) e clique em "Seu negócio"
```
